src/python/issue_management/monorail/__init__.py

```diff
@@ -225,6 +233,42 @@ def get_original_issue(self, issue_id):
     """Retrieve the original issue object traversing the list of duplicates."""
     return Issue(self._itm.get_original_issue(int(issue_id)))
 
+  def find_issues(self, keywords=None, only_open=False):
+    """Find issues."""
+    if not keywords:
+      return None
+
+    search_text = _get_search_text(keywords)
+    if only_open:
+      can = IssueTrackerManager.CAN_OPEN
+    else:
+      can = IssueTrackerManager.CAN_ALL
+
+    issues = self._itm.get_issues(search_text, can=can)
+    return [Issue(issue) for issue in issues]
+
+  def find_issues_url(self, keywords=None, only_open=None):
+    """Find issues (web URL)."""
+    if not keywords:
+      return None
+
+    search_text = _get_search_text(keywords)
+    if only_open:
+      can = IssueTrackerManager.CAN_OPEN
+    else:
+      can = IssueTrackerManager.CAN_ALL
+
+    can_id = IssueTrackerManager.CAN_VALUE_TO_ID_MAP.get(can, '')
+    return ISSUE_TRACKER_SEARCH_URL.format(
+        project=self.project) + urllib.parse.urlencode({
```

Comment:
add format identifier for url params in ISSUE_TRACKER_SEARCH_URL, so that urllib.parse.urlencode can be passed in format itself.

Reply:
Done!